methods/getMailObject.js

```diff
@@ -10,16 +10,18 @@ const schema = Joi.object().keys({
   text: Joi.optional()
 });
 module.exports = function(details, content, allDone) {
-  let from = details.from;
+  let from = details.fromEmail;
```

Comment:
why did this change?  this will break things. Can we check for from and then fall back to fromEmail?